ngraph_creator/operations/include/OperationsBase.hpp

```diff
@@ -40,8 +40,11 @@ class OperationsBase {
         if (sModelInfo->isOperandLifeTimeConst(operandIndex)) {
             auto operandValues = sModelInfo->GetConstVecOperand<T>(operandIndex);
             auto operandDims = getInputOperandDimensions(inputIndex);
-            return std::make_shared<ngraph::opset3::Constant>(
-                ngraph::element::f32, toNgraphShape(operandDims), operandValues);
+            if (operandDims[0] != 0)
```

Comment:
Shouldn't the lifetime be NO_VALUE when operandDims has 0 value?
Is it an error in the vts code ?